### PR TITLE
Implement `NSSet#description`

### PR DESCRIPTION
**Pull request by rikusouda**
Because `NSSet#description` was not implemented.

### CODE REVIEW

**Comment by spevans** [comment]
@swift-ci test

File: Foundation/NSSet.swift [hunk 1]
```diff
@@ -107,9 +107,46 @@ open class NSSet : NSObject, NSCopying, NSMutableCopying, NSSecureCoding, NSCodi
...
+            } else {
+                descriptions.append("\(obj)")
+            }
+        }
+        var indent = ""
+        for _ in 0..<level {
+            indent += "    "
+        }
+        var result = indent + "{(\n"
+        let cnt = count
```
Comment (spevans) [inline Foundation/NSSet.swift]: Is `cnt` needed here? It seems to just mirror the value held in `count`?

**Comment by spevans** [comment]
@swift-ci test

File: Foundation/NSSet.swift (hunk 1 above)
Comment (rikusouda) [reply Foundation/NSSet.swift]: Thanks!!
I fixed it. 231c8be1d55e688ee6b893df2b2c339ee7aaa7c9

**Comment by spevans** [comment]
@swift-ci test

**Comment by ikesyo** [comment]
@swift-ci test Linux platform

**Comment by rikusouda** [comment]
Could you start test? 🙇

**Comment by spevans** [comment]
@swift-ci test

**Comment by ikesyo** [comment]
@swift-ci test

**Comment by rikusouda** [comment]
Fixed it again

**Comment by ikesyo** [comment]
@swift-ci Please clean test Linux platform

**Comment by ikesyo** [comment]
@swift-ci Please clean test

**Comment by ikesyo** [comment]
@swift-ci Please test

**Comment by spevans** [comment]
@swift-ci Please test

**Comment by spevans** [comment]
FYI, the test failures are not related to your changes, they look to be due to issues in one of the other repositories.

**Comment by spevans** [comment]
@swift-ci test linux

**Comment by rikusouda** [comment]
> FYI, the test failures are not related to your changes, they look to be due to issues in one of the other repositories.

Thanks for your information! I got it.

**Comment by rikusouda** [comment]
Is there any action I should to do?

**Comment by ikesyo** [comment]
@swift-ci Please test Linux platform

**Comment by ikesyo** [comment]
Thanks for working on it!

**Comment by rikusouda** [comment]
Thanks a lot

**Pull request merged**